accounts.go

```diff
@@ -40,9 +40,13 @@ type Account struct {
 
 type GetAccountInput struct{}
 
-func (client *AccountsClient) GetAccount(input *GetAccountInput) (*Account, error) {
+func (client *AccountsClient) GetAccount(ctx context.Context, input *GetAccountInput) (*Account, error) {
+	if ctx == nil {
+		panic(nilContext)
```

Comment:
What's special about this particular API endpoint with respect to panicking if the user passes a `nil` Context? Panicking here seems like it would be a surprise to the user and we're not doing it anywhere else. Maybe a default `context.Background` would be more appropriate?